var/spack/repos/builtin/packages/quantum-espresso/package.py

```diff
@@ -367,7 +367,8 @@ class QuantumEspresso(CMakePackage, Package):
     )
 
     # Configure updated to work with AOCC compilers
-    patch("configure_aocc.patch", when="@6.7:6.8 %aocc")
+    patch("configure_aocc.patch", when="@6.7:7.0 %aocc")
+    patch("configure_qe-7.1_aocc.patch", when="@7.1: %aocc")
```

Comment:
Did you upstream this patch?

Reply:
No, this patch has not been upstreamed to QE codebase yet.

Reply:
Can you please submit it, so we don't have to rely on patches on the next version?